apps/avatax/package.json

```diff
@@ -97,7 +98,7 @@
   },
   "private": true,
   "saleor": {
-    "schemaVersion": "3.18"
+    "schemaVersion": "3.20"
```

Comment:
Schema version is not the same thing as required Saleor version. Using new schema only gives us access to new fields, but we don't have to consume them.

I updated schema because I need fields added in 3.19 and 3.20. 
Indeed manifest must be updated, because it points to 3.18. I will create separate ticket to update it to 3.19 as minimum version 👍